canonical/istio-beacon-k8s-operator
# Lib work

## Issue
Update the library to provide the current desired interface and functionality.


## Solution
Fully implement cross model support and simplify the service_mesh interface.


## Testing Instructions
 * Deploy beacon
 * Deploy  https://github.com/dstathis/mesh-tester
 * Repeat the previous step as much as you want
 * Create various relations between the tester charms
 * Relate one of the tester charm to beacon
 * Check that `jhack show-relation beacon tester` has appropriate data
